gene/etl/ensembl.py

```diff
@@ -19,7 +19,7 @@
 class Ensembl(Base):
     """ETL the Ensembl source into the normalized database."""
 
-    def __init__(self, database: Database, host="ftp.ensembl.org",
+    def __init__(self, database: AbstractDatabase, host="ftp.ensembl.org",
```

Comment:
Docstring has type `Database` (same for the other sources). I think we're now getting in the habit of not including the type in the dosctring so this can be done in a separate issue to be consistent throughout